### PR TITLE
docs: updated coverpage to embed video and why-kpt

### PR DESCRIPTION
**Pull request by droot**
This PR makes following changes to the `kpt.dev` homepage:
 - Shows how to embed demo video
 - Add `Why kpt`
 - Changes the text for `Learn more` to `Get Started`

image

### CODE REVIEW

**Comment by bgrant0607** [comment]
@droot Let's go ahead and add the first demo video. https://www.youtube.com/watch?v=L_x7z4CXHDw

**Comment by droot** [comment]
@bgrant0607 updated it. This is how it looks.

image

File: site/coverpage.md [hunk 1]
```diff
@@ -6,4 +6,9 @@
 > by manipulating declarative Configuration as Data, 
 > separated from the code that transforms it.
 
-[Learn More](?id=overview)
+<iframe id="ytplayer" type="text/html" width="550" height="360"
+    src="https://www.youtube.com/embed/L_x7z4CXHDw"> </iframe>
+
+
+[Why kpt](/guides/rationale.md)
```
Comment (selfmanagingresource) [inline site/coverpage.md:13]: Should this be ./? Link preview had / go back to kpt/tree rather than be relative to site directory

**Comment by bgrant0607** [comment]
Could we also add a link to the channel somewhere?
https://www.youtube.com/channel/UCE6EvJ37y6Z8Xu_cDykhlnw

**Comment by bgrant0607** [comment]
Thanks. Video LGTM

**Review by bgrant0607 — APPROVED** [review]

**Comment by droot** [comment]
> Could we also add a link to the channel somewhere?

For now, added twitter and youtube channel link on the contact-us page.

image

**Review by mengqiy — APPROVED** [review]

**Pull request merged**